public/docs/_examples/displaying-data/dart/web/main.dart

```diff
@@ -0,0 +1,16 @@
+// #docregion
+import 'package:angular2/bootstrap.dart';
+import 'package:displaying_data/app_ctor_component.dart';
+import 'package:displaying_data/app_component_1.dart' as v1;
+import 'package:displaying_data/app_component_2.dart' as v2;
+import 'package:displaying_data/app_component_3.dart' as v3;
+import 'package:displaying_data/app_component_final.dart' as v4;
+
+main() {
+//  bootstrap(v1.AppComponent);
+//  bootstrap(v2.AppComponent);
+//  bootstrap(v3.AppComponent);
+  bootstrap(v4.AppComponent);
+  bootstrap(AppCtorComponent);
```

Comment:
removing reference to ctor in `main.dart` but leaving the lib file just in case we want it later. Let me know if I should so otherwise.